src/material/chips/chip-option.spec.ts

```diff
@@ -294,6 +304,20 @@ describe('MDC-based Option Chips', () => {
 
         expect(primaryAction.getAttribute('aria-disabled')).toBe('true');
       });
+
+      it('should display checkmark graphic by default', () => {
+        expect(
```

Comment:
I think that we should remove this assertion since it's basically just checking that the DI system behaves as expected. The two below that check against the DOM are enough.